docs/topics/media-pipeline.rst

```diff
@@ -197,6 +197,25 @@ For self-hosting you also might feel the need not to use SSL and not to verify S
     AWS_USE_SSL = False # or True (None by default)
     AWS_VERIFY = False # or True (None by default)
 
+
+Custom Amazon S3 headers in FileStorage
+
+Steps to pass **SSEKMSKeyId** and **ServerSideEncryption** to **scrapy** FilesPipeline using **AWS S3**
+
+Configuration Steps::
+
+    1. Subclass ``scrapy.pipelines.files.S3FilesStore``, 
+
+    2. Extend its ``HEADERS`` class attribute in your subclass to define the headers you want with the values you 
+    want, for eg, ``X-Amz-Server-Side-Encryption``, ``X-Amz-Server-Side-Encryption-Aws-Kms-Key-Id``, etc.
+
+    3. You can see the header-to-key mapping in the source code of the class, for additional header names,
```

Comment:
This sounds like you can only use one of the headers of the mapping. Maybe it would be good to make that clear if so, and maybe even copy that mapping here, or link somewhere else where it can be seen (botocore documentation?).

Reply:
I couldn't find specific documentation on **Request & Response Headers** in the **Botocore documentation**, but I've referred to the **Botocore S3 documentation** and **AWS documentation** for the reference of the user